string.c

```diff
@@ -4133,10 +4133,13 @@ rb_str_hash_cmp(VALUE str1, VALUE str2)
  * call-seq:
  *   hash -> integer
  *
- * Returns the integer hash value for +self+.
- * The value is based on the length, content and encoding of +self+.
+ * Returns the integer hash value for self.
```

Comment:
I think the documentation uses `+self+` for consistency:

```suggestion
 * Returns the integer hash value for +self+.
```